packages/block-library/src/embed/embed-placeholder.native.js

```diff
@@ -8,13 +8,40 @@ import { View, Text, TouchableWithoutFeedback } from 'react-native';
  */
 import { __ } from '@wordpress/i18n';
 import { usePreferredColorSchemeStyle } from '@wordpress/compose';
+import { useState } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import {
+	useBlockEditContext,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
+import EmbedBottomSheet from './embed-bottom-sheet';
 import styles from './styles.scss';
 
-const EmbedPlaceholder = ( { icon, label, onFocus } ) => {
+const EmbedPlaceholder = ( {
+	icon,
+	isSelected,
+	label,
+	onFocus,
+	value,
+	onSubmit,
+} ) => {
+	const { clientId } = useBlockEditContext();
+	const { wasBlockJustInserted } = useSelect(
+		( select ) => ( {
+			wasBlockJustInserted: select(
+				blockEditorStore
+			).wasBlockJustInserted( clientId, 'inserter_menu' ),
+		} ),
+		[ clientId ]
+	);
+	const [ isEmbedSheetVisible, setIsEmbedSheetVisible ] = useState(
+		isSelected && wasBlockJustInserted && ! value
```

Comment:
This is the condition to determine if the embed sheet should be automatically displayed when the component is mounted.

Reply:
FWIW, verified that the bottomsheet is not triggered if we duplicate the block (via the block's toolbar 3-dot menu).